### PR TITLE
docs(adr): update Remix Icon project activity

### LINKED ISSUE #767: Update ADR 0022 to reflect Remix Icons project being active again
Thanks @pdesoyres-cc for pointing that! :eagle:

### PR DESCRIPTION
**Pull request by roberttran-cc**
Fixes #767 

:sweat_smile: 

A simple documentation update that should be straight-forward to review.

### CODE REVIEW

File: docs/adr/adr-0022-new-icon-system.md [hunk 1]
```diff
@@ -242,6 +242,8 @@ This simplifies the API for developers because:
 
 Where Remix Icon is a great library, it seems to be currently on [pause](https://github.com/Remix-Design/RemixIcon/issues/232).
 
+_edit (2023-09-15)_: actually, the Remix Icon project [resumed its activity](https://github.com/Remix-Design/RemixIcon/issues/232#issuecomment-1017140063) since early 2023! 🎉
```
Comment (Galimede) [inline docs/adr/adr-0022-new-icon-system.md:245]: question: I was wondering if this link was more appropriate: `https://github.com/Remix-Design/RemixIcon/issues/232#issuecomment-1492911872`

**Review by Galimede — APPROVED** [review]
LGTM! :wink:

**Review by florian-sanders-cc — APPROVED** [review]
LGTM!

**Review by pdesoyres-cc — APPROVED** [review]
LGTM

File: docs/adr/adr-0022-new-icon-system.md (hunk 1 above)
Comment (roberttran-cc) [reply docs/adr/adr-0022-new-icon-system.md:245]: Thanks both of you! I'll leave it as is for now. :pray:

**Pull request merged**